app/presenters/placeholder_content_item_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class PlaceholderContentItemPresenter
```

Comment:
I named it this mainly for consistency with the existing `ContentItemPresenter`. i.e. to make it obvious they are related.